src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/FixAllCodeRefactoringSuggestedAction.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixesAndRefactorings;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.UnifiedSuggestions.UnifiedSuggestedActions;
+using Microsoft.VisualStudio.Text;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
+{
+    /// <summary>
+    /// Suggested action for fix all occurrences for a code refactoring.  Note: this is only used
+    /// as a 'flavor' inside CodeRefactoringSuggestionAction.
+    /// </summary>
+    internal sealed class FixAllCodeRefactoringSuggestedAction : FixAllSuggestedAction, IFixAllCodeRefactoringSuggestedAction
+    {
+        internal FixAllCodeRefactoringSuggestedAction(
```

Comment:
1. nit: public.  our general pattern is htat we don't use internal for members unless they're in a public type.
2. FixAllCodeFixSuggestedAction exposed the original diagnostic.  Is nothing like that needed for FixAllCodeRefactoringSuggestedAction?

Reply:
> nit: public. our general pattern is htat we don't use internal for members unless they're in a public type.

This is existing code, but I have still fixed it up. I think this is a good candidate for an analyzer in Roslyn.Diagnostics.Analyzers to enforce such a pattern.

> FixAllCodeFixSuggestedAction exposed the original diagnostic. Is nothing like that needed for FixAllCodeRefactoringSuggestedAction

Nope, its not needed. Diagnostic is required by FixAllCodeFixSuggestedAction for implementation of `ITelemetryDiagnosticID<string>`